services/uploads/serverless.yml

```diff
@@ -109,6 +107,9 @@ functions:
     layers:
       - !Ref ClamAvLambdaLayer
       - arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-nodejs-amd64-ver-1-18-1:1
+    vpc:
+      securityGroupIds: ${self:custom.sgId}
+      subnetIds: ${self:custom.privateSubnets}
```

Comment:
I think you can remove the avDownloadDefinitions lambda, too

Reply:
I wrote a ticket to do this cleanup as follow on work: https://jiraent.cms.gov/browse/MCR-3954

I started deleting the now unneeded code paths and then realized that there's a decent amount in there and the tests are going to have to get fixed up a bit. Figured I'd land this and then go through and delete/cleanup the tests.